### DISCUSSION TITLE
strange symbol in CMD

### DISCUSSION THREAD

**Discussion by wnunezc** [symfony/symfony#44381] [Q&A] [answered]
is normal this strange symbol in CMD/Windows 10:

![image](https://user-images.githubusercontent.com/1207306/144158348-eef1be6a-0d63-44de-904c-30015710ff6b.png)

**Comment by michaoj** [comment, marked as answer]
Looks like your terminal/cmd doe not interpret ansi color codes

**Reply by wnunezc** [reply]
> Looks like your terminal/cmd doe not interpret ansi color codes

you have see the entire capture????
